src/runtime/src/value_map.rs

```diff
@@ -184,6 +184,25 @@ pub enum Operator {
     NotEqual,
 }
 
+/// Get string representation of [Operator].
+pub fn operator_as_string(op: &Operator) -> String {
```

Comment:
This could be an implementation of `Display` trait, but I saw you used this approach for `Value`, so I chose the same way.

Reply:
Ah, `type_as_string` is there for when you're only interested in the type, but there is actually an implementation of `Display` for `Value`. I think in this case `Display` would be more appropriate, sorry that you were misled by the lack of an explanatory comment! I'm aiming to do a documentation sweep as soon as possible.

Reply:
OK, I'll make a change.